src/HighlightAuto.svelte

```diff
@@ -57,7 +56,7 @@
     position: absolute;
     top: 0;
     right: 0;
-    padding: calc(1em + 3px) calc(1em + 5px);
+    padding: 1em;
```

Comment:
I'd prefer to use `rem` over `em`. Any specific reason for using `em`?

Reply:
I'd also prefer `rem`, but for consistency with `highlight.js` I'm using `em`;
```css
pre code.hljs {
  display: block;
  overflow-x: auto;
  padding: 1em; /**/
}
```
*snippet showing use of `em` for padding, present in all of `highlight.js`'s stylesheets*

This prevents over-sizing or under-sizing of the langtag relative to the code block when the `rem` and `em` differ. This is especially notable on single-line code blocks, and though it may be somewhat edge-case I feel it's worth supporting for how simple it is to do so.

We can change this to `rem` if you would like.

Reply:
Makes sense – let's keep `em` in that case